### PR TITLE
Milestone 2: Add Functional Components

### PR DESCRIPTION
**Pull request by cherelemma**
# Implementation Summary

In this Math Magicians project milestone, the following project requirements are implemented.

- Delete all the boilerplate from CRA (text, images, styles)
- Create a directory called components.
- Create a React component that matches the given calculator design.
- Add a style and format the calculator layout.

## General requirements checklist.

✅  No linters error.
✅  Followed the GitHub flow.
✅  Documented work professionally.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+function Calculator() {
```
Comment (victorokoroji) [inline src/components/Calculator.js]: - [ ] Good job adding the calculator component 👍🏽 . Please ensure the `Calculator.js` component is written in a class-based component as specified in the project requirement 👍🏽.

File: src/index.js [hunk 2]
```diff
@@ -1,17 +1,11 @@
...
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import './style.css';
+import Calculator from './components/Calculator';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
-    <App />
+    <Calculator />
```
Comment (victorokoroji) [inline src/index.js]: - [ ] I will recommend you don't delete the App.js file but rather the `App.js`(main component) should have the imported `Calculator.js` component and it should be a class-based component 👍🏽

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi @cherelemma, 

You have done a great job so far! 💪 👏
You still need to work on a few issues before moving on to the next project, but you're almost there!

## Top highlight 🎯

- The Pull Request is following the Git Flow ✅ 
- The PR title and description are professional and descriptive ✅  
- Linters are working properly ✅ 


## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by GzimAsani — APPROVED** [review]
# STATUS: APPROVED :100:         
## There is nothing else to say than it's time to merge... :partying_face: 
Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


## Optional suggestions


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- NA
### Please be reminded to not open a new pull request after you get a review, only use the same one :thought_balloon: 


 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**